### PR TITLE
Fix zinolib annoyances

### PR DESCRIPTION
**Pull request by hmpf**
Best read commit by commit

### CODE REVIEW

**Review by lunkwill42 — CHANGES_REQUESTED** [review]
Mostly good to go, but naming things is hard and I'm not entirely convinced (yet)...

File: src/zinolib/ritz.py [hunk 1]
```diff
@@ -399,7 +399,7 @@ def __del__(self):
         """Zino object deletion"""
         self.close()
 
-    def _read_command(self, command, recv_buffer=4096, delim="\r\n"):
+    def _request(self, command, recv_buffer=4096, delim="\r\n"):
```
[the remark this answers is not in the document]
Comment (hmpf) [reply src/zinolib/ritz.py]: It doesn't read commands, it *sends* commands, reads the response, and returns the response. Statistically it seems to me this is usually named as a "request" that returns a "response", and now this code follows that de facto rule as well.

[the remark this answers is not in the document]
Comment (lunkwill42) [reply src/zinolib/ritz.py]: Ah indeed, this is the function you were telling me about a while back! Thanks!

**Review by lunkwill42 — APPROVED** [review]
Alright, I'm convinced 👍

**Comment by hmpf** [comment]
I'd like comments from @runborg as well before I merge.

**Review by runborg — APPROVED** [review]
As for the naming of _read_command, the original name is not a good name for it.. the new purposed name is way better :)

For the other changes they seems fine, no big comments.. :)

**Pull request merged**